From c3e34a8fb45bd5d1915d940954ddebdf09f4b3a2 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Thu, 2 Jan 2025 07:47:10 +0000
Subject: [PATCH] Updating graphics lib to v4 for latest code

Latest AOSP uses v4 instead of v3. Updating the same

Tests Done: Build and boot

Tracked-On: OAM-128386
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 hwc3-server/default/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwc3-server/default/Android.bp b/hwc3-server/default/Android.bp
index 8505248..c6eaa4d 100644
--- a/hwc3-server/default/Android.bp
+++ b/hwc3-server/default/Android.bp
@@ -54,7 +54,7 @@ cc_binary {
     shared_libs: [
         "android.hardware.graphics.composer@2.1-resources",
         "android.hardware.graphics.composer@2.2-resources",
-        "android.hardware.graphics.composer3-V3-ndk",
+        "android.hardware.graphics.composer3-V4-ndk",
         "libbase",
         "libbinder_ndk",
         "libcutils",
-- 
2.34.1

